prosemirror-jdita/tsconfig.json

```diff
@@ -1,69 +1,19 @@
 {
-  "exclude": [
-    "**/*.spec.ts",
-    "src/tests.ts"
-  ],
+  "extends": "@tsconfig/recommended/tsconfig.json",
+
+  // overrides recommended
   "compilerOptions": {
-    /* Basic Options */
-    // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es6",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */
-    // "lib": [],                             /* Specify library files to be included in the compilation. */
-    // "allowJs": true,                       /* Allow javascript files to be compiled. */
-    // "checkJs": true,                       /* Report errors in .js files. */
-    // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
-    "declaration": true,                   /* Generates corresponding '.d.ts' file. */
-    // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
+    "target": "es6",                          /* Target ECMAScript version 6. */
```

Comment:
@tsconfig/recommended/tsconfig.json specifies the target as `ES2016` I think we should stick with that.
Check: https://www.npmjs.com/package/@tsconfig/recommended.